src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const handleTabClick = (tabId, event) => {
+    event.preventDefault();
+    if (activeTabId !== tabId) {
+      onTabSelected(tabId);
+    }
+  };
+
+  const activeTabContent = tabs.find(({ id }) => id === activeTabId)?.content;
```

Comment:
also redundant

Reply:
Why?
Mattnor said to take the logic out of the way
How will it work then?